RadixWallet/Clients/OnLedgerEntitiesClient/Helpers/OnLedgerEntitiesClient+ComplexTypes.swift

```diff
@@ -119,7 +143,11 @@ extension OnLedgerEntitiesClient {
 					details: .init(
 						address: poolContribution.poolAddress,
 						dAppName: resourceAssociatedDapps?[resourceAddress]?.name,
-						poolUnitResource: .init(resource: resource, amount: .init(nominalAmount: amount)),
+						poolUnitResource: .init(
+							resource: resource,
+							amount: .exact(.init(nominalAmount: amount)),
```

Comment:
seeing this multiple times, maybe worth declaring a function on ResourceAmount like:
```swift
static func exact(_ nominalAmount: Decimal) -> Self
```